lms/djangoapps/teams/tests/test_views.py

```diff
@@ -2657,3 +2657,78 @@ def test_upload_learners_exceed_max_team_size(self):
         )
         response_text = json.loads(response.content.decode('utf-8'))
         self.assertEqual(response_text['errors'][0], 'Team {} is full.'.format(team1))
+
+    def test_deletion_via_upload_csv(self):
+        self.test_upload_valid_csv_simple()
```

Comment:
Add a comment that this is to create certain exisiting memberships
(I think that's why you do this? I would know for sure if there was a comment ;) )